lib/manager/npm/post-update/index.ts

```diff
@@ -170,7 +173,7 @@ export async function writeExistingFiles(
         logger.warn({ yarnrcFilename, err }, 'Error writing .yarnrc');
       }
     }
-    const { npmLock } = packageFile;
+    const { npmLock } = (packageFile.managerData || {}) as NpmManagerData;
```

Comment:
```suggestion
    const { npmLock } = packageFile?.managerData;
```

Reply:
I had this guard partly just in case `managerData` was empty